config/formats.js

```diff
@@ -3420,6 +3426,20 @@ exports.Formats = [
 					};
 				}
 			}
+			if (move.id === 'highjumpkick' && name === 'galbia') {
+				move.name = 'Kibitz';
+				move.basePower = 110;
+				move.accuracy = 100;
+				delete move.onMoveFail;
+				move.onHit = function (target, source) {
+					var result = this.random(100);
+					if (result < (pokemon.ability === 'serenegrace' ? 60 : 30)) {
+						target.addVolatile('flinch');
+					} else if (target.hp !== 0) {
+						this.damage(source.maxhp / 3, source, source, 'Kibitz');
```

Comment:
Is the idea that %galbia takes damage if he doesn't flinch the opponent? If so, how will you deal with the case when he moves after the opponent?